napbot/extensions/music/__init__.py

```diff
@@ -44,6 +59,8 @@ def __init__(self, bot: commands.Bot):
                 conf.getfloat("GuessVoteSkipPercent", 0.0) / 100
             )
             self.guess_lenient: bool = conf.getboolean("GuessLenient", fallback=True)
+            self.temp_folder: Path = Path(tempfile.mkdtemp(dir=Path(conf.get("TempPath", fallback="/tmp"))))
```

Comment:
Use `tmpfile.TemporaryDirectory()` and call `temp_folder.cleanup()` instead of manually doing this

You also don't need to leave this configurable bc it finds a directory for you (%APPDATA%/Temp/blah on windows, /tmp/blah on macos and linux)